src/main/java/uk/gov/ons/ctp/response/casesvc/domain/model/Case.java

```diff
@@ -88,9 +88,12 @@
   @Column(name = "collectioninstrumentid")
   private UUID collectionInstrumentId;
 
-  @Column(name = "actionplanid")
+  @Column(name = "actionplanid", nullable= true)
   private UUID actionPlanId;
 
+  @Column(name = "active_enrolment")
```

Comment:
Does this element also need to be nullable?  What happens if this isn't included (like it is in the current version with the actionplanid)

Reply:
Should default to false as its boolen